docs/dev/tutorials/api3-usd-paymaster-tutorial.md

```diff
@@ -7,6 +7,10 @@ head:
 
 # USDC paymaster tutorial with API3 dAPIs
 
+:::Note
+The team is migrating to the Sepolia testnet, and for this tutorial, we're updating the contract addresses for `ETHUSDdAPI` and `USDCUSDdAPI` with the API3 team. Only then will the tutorial be functional in the zkSync Sepolia testnet. For now, it's available only with the Goerli testnet.
+:::
```

Comment:
I'd rather create a tip warning that specifies:
```
This tutorial is currently operational exclusively on the Goerli testnet. Stay tuned for upcoming support for the Sepolia network.
```

Much cleaner imo.

Reply:
Yes it is cleaner, thanks for suggesting!